index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="styles.css" />
```

Comment:
Great job organizing your files. However, you still need to update the path of your files in the HTML files. Kindly fix all the paths in your HTML files.
For example, the path in the "href" attribute should be ```href="./stylesheets/style.css"```.